README.md

```diff
@@ -10,6 +10,10 @@ Like the [Amazon Bedrock Audio Summarizer](https://github.com/aws-samples/amazon
 
 Distill CLI supports configurable language codes for transcription using Amazon Transcribe. Specify the language of the audio file with the `--language-code` or `-l` parameter (e.g., `--language-code es-US` for Spanish). If no language code is provided, it defaults to English (United States) (`en-US`). For a list of supported languages and their codes, see the [Amazon Transcribe Supported Languages Documentation](https://docs.aws.amazon.com/transcribe/latest/dg/supported-languages.html). Use `--help` for more information on available options. The prompt in the provided config.toml is adjusted so that the provided output matches the language in the transcript.
 
+# S3 Object Deletion
+
+Distill CLI supports deletion of the audio recodings once transcribe and summarize activities performed with success. By using the `--delete-s3-object` or `-d` parameter (e.g., `--delete-s3-object Y`), the audio file previously uploaded onto Amazon S3 will be deleted. Use `--help` for more information on available options.
```

Comment:
Ugh, I think I misinterpreted how this is working, my apologies. So the way this works that is if you pass `Y` it will delete the resource when complete. However, it defaults to `N` which keeps the resource?

Reply:
To confirm: Do we still require to implement your suggestion on grabbing the S3 Object name (or not)?

Reply:
No, this is fine. Sorry for the confusion.